static/js/dataset-editor.js

```diff
@@ -281,7 +282,7 @@ var ClassList = React.createClass({
                 <h4>Classes</h4>
```

Comment:
React is meant to be used for UI, so I'd say yes. Take a look at https://facebook.github.io/react/docs/thinking-in-react.html, for example.